styles/prism.css

```diff
@@ -1,144 +1,166 @@
 /**
  * CSS Styles for code highlighting.
- * Feel free to customize token styles 
+ * Feel free to customize token styles
  * by copying from a prismjs compatible theme:
  * https://github.com/PrismJS/prism-themes
  */
+/**
+ * MIT License
+ * Copyright (c) 2018 Sarah Drasner
+ * Sarah Drasner's[@sdras] Night Owl
+ * Ported by Sara vieria [@SaraVieira]
+ * Added by Souvik Mandal [@SimpleIndian]
+ */
 
-/* Code title styles */
-.remark-code-title {
-  @apply su-rounded-t su-bg-cool-grey su-px-5 su-py-3 su-font-mono su-font-bold su-text-black-20;
-}
+code[class*="language-"],
+pre[class*="language-"] {
+	color: #d6deeb;
+	background-color:	#011627;
+	font-family: "Roboto Mono", Consolas, Monaco, "Andale Mono", "Ubuntu Mono", monospace;
+	text-align: left;
+	white-space: pre;
+	word-spacing: normal;
+	word-break: normal;
+	word-wrap: normal;
+	line-height: 1.5;
+	font-size: 0.85em;
+  max-height: 40rem;
 
```

Comment:
Pulled in a different css theme for the code block highlight for accessibility (better contrast 😄 )

Reply:
That was on our to-do list, thanks! Updated theme looks great 😄